packages/web/server/api/edge-routers/assets-router.ts

```diff
@@ -34,7 +35,7 @@ const GetInfiniteAssetsInputSchema = InfiniteQuerySchema.merge(
 ).merge(UserOsmoAddressSchema);
 
 export const assetsRouter = createTRPCRouter({
-  getAsset: publicProcedure
+  getUserAsset: publicProcedure
```

Comment:
Given that the osmo address is optional, I think it would be better to use a more general name, such as getAssets. What do you think?